package.json

```diff
@@ -263,5 +265,8 @@
     "Jenkinsfile": "lint:jenkinsfile",
     "*package.json": "lint:package-names"
   },
-  "snyk": true
+  "snyk": true,
+  "devDependencies": {
```

Comment:
Did we ever figure out why we didn't have devDependencies before?

Reply:
It's an sdk so everything is a normal dependency? I should probably move these over as a normal dependency.